zerver/webhooks/github/view.py

```diff
@@ -910,13 +910,24 @@ def api_github_webhook(
     payload: JsonBodyPayload[WildValue],
     branches: str | None = None,
     user_specified_topic: OptionalUserSpecifiedTopicStr = None,
+    skip_private_repositories: str = "0",
```

Comment:
I think this could be marked as an integer?

Reply:
> Yeah, the main risk is people typing other forms like True, but given we are generating this for them, probably that's OK.

Yes, I have converted this to a boolean as discussed.